File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.py
def solution(A,B):
    answer = 0
    length = len(A)
    A.sort()
    B.sort(reverse=True)
    
    for i in range(length):
        
        num1 = A[i]
        num2 = B[i]
        
        answer += num1 * num2 
        

    return answer

File: 프로그래머스/2/12941. 최솟값 만들기/최솟값 만들기.java
import java.util.*;
class Solution
{
    public int solution(int []A, int []B)
    {
        int answer = 0;
        Integer[] arr = new Integer[B.length];
        for(int i=0; i<arr.length; i++){
            arr[i] = B[i];
        }
        Arrays.sort(A);
        Arrays.sort(arr, Collections.reverseOrder());
        
        for(int i=0; i<A.length; i++){
            answer += A[i] * arr[i];
        } 

        return answer;
    }
}